rules/Proj0003.md

```diff
@@ -0,0 +1,71 @@
+# Proj0003: Define usings explicit
+
+Once enabled, depending on the type of project you have created you'll have the
+following global using statements added to your project implicitly.
+
+| SDK | Default namespaces                       |
+|-----|------------------------------------------|
+| **Microsoft.NET.Sdk**
+|     | System                                   |
+|     | System.Collections.Generic               |
+|     | System.IO                                |
+|     | System.Linq                              |
+|     | System.Net.Http                          |
+|     | System.Threading                         |
+|     | System.Threading.Tasks                   |
+| **Microsoft.NET.Sdk.Web**
+|     | System.Net.Http.Json                     |
+|     | Microsoft.AspNetCore.Builder             |
+|     | Microsoft.AspNetCore.Hosting             |
+|     | Microsoft.AspNetCore.Http                |
+|     | Microsoft.AspNetCore.Routing             |
+|     | Microsoft.Extensions.Configuration       |
+|     | Microsoft.Extensions.DependencyInjection |
+|     | Microsoft.Extensions.Hosting             |
+|     | Microsoft.Extensions.Logging             |
+| **Microsoft.NET.Sdk.Worker**
+|     | Microsoft.Extensions.Configuration       |
+|     | Microsoft.Extensions.DependencyInjection |
+|     | Microsoft.Extensions.Hosting             |
+|     | Microsoft.Extensions.Logging             |
+
+Although this might sound like a good idea, there are downsides. It turns out
+that with multiple targets enabled (frameworks versions mainly) some of the
+inlcuded namespaces (like `System.Net.Https`) are not even available unless
+a NuGet packages included.
+
+But more importantly, it is a lot of magic that can be archived easily without
```

Comment:
But more importantly, it obscures the dependencies of your project. In addition, reducing using statements in your project can be easily achieved by defining namespaces globally in a file.